db/migrate/20160818091421_add_missing_view_permissions2.rb

```diff
@@ -0,0 +1,21 @@
+class AddMissingViewPermissions2 < ActiveRecord::Migration
+  def up
+    default_permissions = Foreman::Plugin.find("foreman_discovery").default_roles
```

Comment:
:+1: running this I found the list was missing view_locations which should be added.